packages/cli/package.json

```diff
@@ -58,67 +64,57 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@babel/preset-env": "^7.3.1",
     "@preact/async-loader": "next",
-    "autoprefixer": "^8.4.1",
+    "autoprefixer": "^9.4.8",
     "babel-esm-plugin": "^0.4.0",
     "babel-loader": "^8.0.5",
-    "babel-plugin-jsx-pragmatic": "^1.0.2",
     "babel-plugin-macros": "^2.4.5",
-    "babel-plugin-transform-export-extensions": "^7.0.0-alpha.19",
-    "babel-plugin-transform-react-remove-prop-types": "^0.4.23",
-    "bluebird": "^3.5.0",
-    "chalk": "^2.4.1",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "console-clear": "^1.0.0",
-    "copy-webpack-plugin": "^4.1.0",
+    "copy-webpack-plugin": "^5.0.0",
     "critters-webpack-plugin": "^1.3.3",
     "cross-spawn-promise": "^0.10.1",
-    "css-loader": "^0.28.7",
-    "css-modules-require-hook": "^4.0.6",
+    "css-loader": "^2.1.0",
     "ejs-loader": "^0.3.0",
-    "esm": "^3.0.84",
+    "esm": "^3.2.5",
     "fast-async": "^6.3.7",
-    "file-loader": "^1.1.11",
-    "fs.promised": "^3.0.0",
-    "get-port": "^3.1.0",
+    "file-loader": "^3.0.1",
+    "get-port": "^4.1.0",
     "gittar": "^0.1.0",
-    "glob": "^7.1.2",
     "html-webpack-exclude-assets-plugin": "0.0.7",
     "html-webpack-plugin": "^3.2.0",
-    "inquirer": "^5.2.0",
+    "inquirer": "^6.2.2",
     "ip": "^1.1.5",
     "isomorphic-unfetch": "^3.0.0",
+    "kleur": "^3.0.1",
     "loader-utils": "^1.1.0",
-    "log-symbols": "^2.1.0",
-    "mini-css-extract-plugin": "^0.4.0",
+    "loglevelnext": "^3.0.0",
```

Comment:
Where does this get used?

Reply:
oh, accidentally added it, will remove it again